librz/main/rz-ax.c

```diff
@@ -736,3 +736,34 @@ RZ_API int rz_main_rz_ax(int argc, const char **argv) {
 	num = NULL;
 	return 0;
 }
+
+
+char* rz_ax_parse_bitstring(const char* bitstring) {
+    // Check that bitstring starts with "0b"
+    if (strncmp(bitstring, "0b", 2) != 0) {
+        RZ_LOG_ERROR("rz-ax: bitstring must start with '0b'\n");
+        return NULL;
+    }
+    // Get the length of the bitstring
+    size_t bitlen = strlen(bitstring) - 2;
+	// Create a bitvector of the appropriate size
```

Comment:
Please use `clang-format` to fix those indentation and space fixes `char *name` instead of `char* name`

Reply:
nono, this is regarding the C code format, not the arithmetic ops in rz-ax